### DISCUSSION TITLE
Installment Lazyvim for non-root user on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Forone-Shijie** [LazyVim/LazyVim#2252] [Q&A] [answered]
Hi, I am very interested in lazyvim recently, and I'm trying to install Lazyvim for  all my used platforms. I have successfully installed it on my macbook as well as my windows laptop, and they worked very well.

However, when I tried to install it on my cloud computer (Ubuntu 20.04), I face some problems.

That is, I have successfully installed neovim through
 ```sudo apt install neovim```
 I used ```whereis nvim``` and ```nvim --version``` to show some details ,and the outputs are as followes:
![image](https://github.com/LazyVim/LazyVim/assets/63043963/1bae87f0-fc36-4425-917c-0315d765eebf)
For configuring the Lazyvim, I followed the tutorial:
```git clone https://github.com/LazyVim/starter ~/.config/nvim```
And I found that succeed, which is could be shown as :
![image](https://github.com/LazyVim/LazyVim/assets/63043963/cee06847-d015-417e-8a2b-0243314b8371)

But when I start the nvim, it could not load the Lazyvim configuration, either do the root user. 
![image](https://github.com/LazyVim/LazyVim/assets/63043963/4723decf-48a4-4b8b-9a47-8ad9cbaef70b)

So I guess there may be some steps I should do to use Lazyvim for myself, which I don't know. I will be greatly appreciated for your help, because I'm really interested in using Lazyvim for my usual work.

**Comment by jyuan0** [comment, marked as answer]
The version of `nvim` in the 20.04 apt sources is too old (v0.4.3). You need Neovim >= 0.9.0 (see the LazyVim [requirements](https://github.com/LazyVim/LazyVim#%EF%B8%8F-requirements)).
I recommend you use [bob](https://github.com/MordechaiHadad/bob) to install `nvim` instead.

**Reply by Forone-Shijie** [reply]
Thank u ! I have sucessfully installed it on my platform. That would be greatly helpful for me.
